app/api/domains/osu.py

```diff
@@ -551,19 +517,19 @@ def handle_invalid_characters(s: str) -> str:
             ],
         )
 
-        ret.append(
-            DIRECT_SET_INFO_FMTSTR.format(
-                Artist=handle_invalid_characters(bmap["Artist"]),
-                Title=handle_invalid_characters(bmap["Title"]),
-                Creator=bmap["Creator"],
-                RankedStatus=bmap["RankedStatus"],
-                LastUpdate=bmap["LastUpdate"],
-                HasVideo=bmap["HasVideo"],
-                diffs=diffs_str,
-                **{DIRECT_SET_ID_SPELLING: bmap[DIRECT_SET_ID_SPELLING]},
-            ),
+        fmstr = DIRECT_SET_INFO_FMTSTR.format(
```

Comment:
Any reason for this change?

Reply:
I don't quite get what you mean

Reply:
You mean that I'm saving it in an extra variable? I think it's just something that happened while debugging, should I revert it?

Reply:
ye, was just curious as to why u changed it.  also i assume set id spelling is gone

Reply:
Yeah chimu had SetId instead of SetID which is a violation of the cheese gull standard and cmyui for some reason rather implemented that than telling kotrik about it but kotrik said he fixed it now

Reply:
> You mean that I'm saving it in an extra variable? I think it's just something that happened while debugging, should I revert it?

can revert that btw